recipe/meta.yaml

```diff
@@ -19,7 +19,8 @@ build:
   number: 4
   # for cuda on win/linux, building with 9.2 is enough to be compatible with all later versions,
   # since arrow is only using libcuda, and not libcudart.
-  skip: true  # [(win or linux) and cuda_compiler_version not in ("None", "10.2")]
+  skip: true  # [(win or linux64) and cuda_compiler_version not in ("None", "10.2")]
+  skip: true  # [linux and (aarch64 or ppc64le) and cuda_compiler_version not in ("None", "11.2")]
```

Comment:
Re-rendering now works, but the needed builds are missing. Think fixing this skip should help. We start with CUDA 11.2 for arch builds